src/app/genome/genome.view.model.ts

```diff
@@ -68,8 +71,13 @@ export default class GenomeViewModel {
         for (let workerModule of workerModules) {
             let workerModuleName = workerModule.module.replace("AseqCompute:", "");
             this.workerModules[workerModuleName] = workerModule.state;
-            if (workerModule.state === "done") 
-                this.workerModulesTooltips[workerModuleName] = this.workerModulesTooltips[workerModuleName].replace("is not complete.", "is complete.");
+            if (workerModule.state === "done")
+            try {
```

Comment:
No need to put this in try/catch block. Please remove. Instead in case in the future new workerModules will be added a save solution is:
`
if (workerModule.state === "done" && this.workerModulesTooltips.hasOwnProperty(workerModuleName))
    this.workerModulesTooltips[workerModuleName] = this.workerModulesTooltips[workerModuleName].replace("is not complete.", "is complete.");
`
Please change to this one.

Reply:
Done in new commit!